wallet/address/address.go

```diff
@@ -55,12 +60,49 @@ func (a *Address) UnmarshalBinary(data []byte) error {
 	return nil
 }
 
-// TODO: This should probably return an address like `addr1vpu5vlrf4xkxv2qpwngf6cjhtw542ayty80v8dyr49rf5eg0yu80w`
 // String returns the public key as hex string.
 func (a Address) String() string {
 	return hex.EncodeToString(a.pubKey[:])
 }
 
+// GetTestnetAddress returns the testnet address string representation of this Address (i.e. `addr_test1...`).
+func (a Address) GetTestnetAddress() (string, error) {
+	const testnetIdentifierByte byte = 0x60
+	const testnetIdentifierString = "addr_test"
```

Comment:
I would create constants for at least the identifier strings. Even better would either be a type which links `bytes <-> string` and use it as a parameter for `convertToAddress`. This could either be a map with accessor functions, or just a simple type `MainnetIdentifier | TestnetIdentifier` which implement a `NetworkIdentifier interface` allowing to access `byte` and `string`. 

But this is up to you (:

Reply:
Done (the first XD)